doc/nvim-tree-lua.txt

```diff
@@ -252,7 +252,7 @@ Here is a list of the options available in the setup call:
 
 *nvim-tree.sort_by*
 - |sort_by|: changes how files within the same directory are sorted. can be
-  one of 'name' | 'modification_time'
+  one of 'name' | 'Name' (case sensitive) | 'modification_time'
```

Comment:
Great idea, but the value is too complex.

How about `name_case_sensitive`

Reply:
The new option that I use is “Name”， which means case sensitive name sorter. 
'name' is still the default option, case insensitive name sorter.
I guess the doc part is misleading, need some updates.